dataland-e2etests/build.gradle.kts

```diff
@@ -38,7 +38,7 @@ dependencies {
 }
 
 tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
-    dependsOn("generateBackendClient")
+    dependsOn("generateBackendClient", "getTestData")

```

Comment:
Wieso brauchen wir das jetzt auf einmal?

Reply:
Mich hatte es beim testen letzte Woche genervt, dass der TestDataProvider manchmal gemeckert hat, die JSON Files, die ich ihm geben möchte, existieren nicht.
Wenn ich vor dem Kompilieren jedoch dafür sorge, dass die JSONs in den korrekten Ordner kopiert werden über die `getTestData` Task, dann habe ich das Problem nicht mehr.